themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```diff
@@ -12,6 +13,10 @@
   top: -1px;
   white-space: nowrap;
 
+  &--hide {
+    display: none;
+  }
```

Comment:
Alternatively, you can use the `hidden` attribute.

Reply:
Set this to `visibility: hidden`